.github/workflows/UpdatePropertiesTableInMSDocs.yaml

```diff
@@ -0,0 +1,99 @@
+# This workflow will create PR to update the configuration properties files in MS docs repository: https://github.com/MicrosoftDocs/azure-dev-docs-pr/tree/main/articles/java/spring-framework/includes/spring-cloud-azure
+# For example, this PR is created by this workflow: https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3035
+
+name: Update the configuration properties files in MS docs repo
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  update-change:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build latest azure-sdk-for-java repository
+        run: |
+          git init azure-sdk-for-java
+          cd azure-sdk-for-java
+          git remote add origin https://github.com/Azure/azure-sdk-for-java.git
+          git config core.sparsecheckout true
+          echo "sdk/spring" >> .git/info/sparse-checkout
+          echo "eng" >> .git/info/sparse-checkout
+          echo "sdk/keyvault" >> .git/info/sparse-checkout
+          echo "sdk/boms" >> .git/info/sparse-checkout
+          echo "sdk/aot" >> .git/info/sparse-checkout
+          git pull --depth=1 origin main
```

Comment:
Is it possible to use this instead? https://github.com/snow-actions/sparse-checkout
You can use another PR to do this.

Reply:
I will investigate.